tests/cypress/e2e/jcontent/uploadMedia.cy.ts

```diff
@@ -7,14 +7,14 @@ describe('Upload media tests', {numTestsKeptInMemory: 1}, () => {
     const siteKey = 'uploadMediaSite';
     const user = {name: 'editoruser', password: 'password'};
 
-    before(function () {
+    beforeEach(function () {
```

Comment:
using `beforeEach`/`afterEach` instead of `before`/`after` as the same files are being uploaded (drag & drop vs dialog) so it's simpler to start from an empty site each time.
But if that's a concern in terms of execution time, I can change it

Reply:
Seems to be still working so no issues for me

Reply:
Good idea, will do, thanks!